lib/dbPool.js

```diff
@@ -311,40 +167,43 @@ class DBPool {
    * @returns {DBPoolConnection} - one connection from the DBPool.
    */
   attach() {
-    let me = this,
-      validConnection = false,
-      connections = me.connections,
-      connectionsLength = connections.length,
-      incrementSize = me.incrementSize,
-      connection,
-      i,
-      j;
-
-    me.log(`Finding available Connection...`);
+    const me = this;
+    const { connections } = me;
+    const { incrementSize } = me;
```

Comment:
This and the line above can be done in one line.
```
const { connections, incrementSize } = me;
```

Reply:
Thanks! will add this change